testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -1,10 +1,12 @@
 # Copyright (c) 2019-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
+#
 # This feature is a dependency for:
-# - features/secondary/min_monitoring.feature : If this feature fails could let monitoring feature disabled for SLE minion
-# - features/secondary/min_centos_monitoring.feature : If this feature fails could let monitoring feature disabled for CentOS minion
+# - features/secondary/min_monitoring.feature: if this feature fails, it could let the monitoring feature disabled for SLE minion
+# - features/secondary/min_rhlike_monitoring.feature: if this feature fails, it could let the monitoring feature disabled for Red Hat-like minion
```

Comment:
```suggestion
# - features/secondary/min_monitoring.feature: if this feature fails, it could let the monitoring feature disabled for the SLE minion
# - features/secondary/min_rhlike_monitoring.feature: if this feature fails, it could let the monitoring feature disabled for the Red Hat-like minion
```

Reply:
yes too

Reply:
done too